code/datums/wounds/pierce.dm

```diff
@@ -89,13 +89,13 @@
 	else if(I.tool_behaviour == TOOL_CAUTERY || I.get_temperature())
 		tool_cauterize(I, user)
 
-/datum/wound/pierce/on_xadone(power)
+/datum/wound/pierce/on_xadone(power, delta_time, times_fired)
 	. = ..()
-	blood_flow -= 0.03 * power // i think it's like a minimum of 3 power, so .09 blood_flow reduction per tick is pretty good for 0 effort
+	blood_flow -= 0.03 * power * delta_time // i think it's like a minimum of 3 power, so .09 blood_flow reduction per tick is pretty good for 0 effort
 
 /datum/wound/pierce/on_synthflesh(power)
 	. = ..()
-	blood_flow -= 0.05 * power // 20u * 0.05 = -1 blood flow, less than with slashes but still good considering smaller bleed rates
+	blood_flow -= 0.025 * power // 20u * 0.05 = -1 blood flow, less than with slashes but still good considering smaller bleed rates
```

Comment:
Why do these two deal with the delta time conversion differently? The result of how they're used?

Reply:
I think I just forgot the delta_time for one of them.

Reply:
Done. I moved the delta time out of the x-adone proc.